config/flags.go

```diff
@@ -25,7 +25,8 @@ const (
 	StorageFlagsCategory    = "Storage"
 	RedisCategory           = "Redis Cache/Fallback"
 	S3Category              = "S3 Cache/Fallback"
-	VerifierCategory        = "KZG and Cert Verifier"
+	VerifierCategory        = "Cert Verifier"
```

Comment:
```suggestion
	VerifierCategory        = "Cert Verifier (V1 only)"
```

Reply:
can we add something to show that this is V1 only. I would even be tempted to add to the KZG category that its shared between V1 and V2.

Reply:
[4e6ea09](https://github.com/Layr-Labs/eigenda-proxy/pull/344/commits/4e6ea09a7a6f352e6cc80a76ce0cb9aac41b8737)

> I would even be tempted to add to the KZG category that its shared between V1 and V2

I don't think it makes sense to specify this for the KZG category. As it now stands with the commit above, _every_ category that doesn't have v1 or v2 specified is for both versions.